### DISCUSSION TITLE
Suggestion: Sokol GH-org

### DISCUSSION THREAD

**Discussion by kassane** [kassane/sokol-d#36] [Ideas]
@floooh, have you ever thought about centralize sokol projects in a GH organization?

**Comment by floooh** [comment]
From time to time, but I'm not a big fan. We did something similar here: https://github.com/fips-libs but I really can't identify with that, it's so 'corporate-like' ;)

For the time being I think it's fine to have a decentralized approach. Can use the sokol-readme to link to related projects.

**Reply by kassane** [reply]
All right!
The treatment organization in GH is tedious. In that case, sokol-d and sokol-`swift 6`(priv, wip) should be transferred to the organization.

**Comment by kassane** [comment]
@floooh, I've requested you take over the project.
I need a break from developing new projects, but I'll send you a functional project as promised.

For dub-registry access, check:
- https://github.com/bdwgc/bdwgc-d/issues/9#issuecomment-3362586427

**Comment by floooh** [comment]
Huh? How is this related to sokol-d?

**Reply by kassane** [reply]
More or less.

The mentioned link is similar to the sokol-d case.
However, the sokol-d transfer is user-to-user without GH-org.

Check your email. There will likely be a request to transfer the project to your user.

**Comment by floooh** [comment]
Hmm ok, I won't be able to do any actual maintenance work or do releases, but I can try to keep the current state working with the automatic bindings updates.

**Reply by kassane** [reply]
The release is very simple: make new tags version (update `dub.json` version too), and dub-registry will automatically detect them. I have changed the repository owner, and you can update without problems.
For maintenance, mention me and I'll provide patches asap.

I'm going offline for a while due to changes in my job, but I'll check some notifications on the GH-mobile app.
